### ISSUE TITLE
Data missing

### ISSUE THREAD

**Issue by Zinference** [deependersingla/deep_portfolio#5]
hello, thanks for you code, but I just wonder that where is the data, which you  mean "/data/oneminutedata"

**Comment by lefig** [comment]
Exactly, there is no use in posting this code without the data that it depends upon.

Given the time and no update to the above comment this appears unlikely.

How unfortunate.
